test/specmap/refs.js

```diff
@@ -467,6 +508,86 @@ describe('refs', () => {
           },
         };
 
+        return mapSpec({
+          spec: input,
+          plugins: [refs],
+        }).then((res) => {
+          expect(res.spec).toEqual(input);
+          expect(res.errors).toEqual([]);
+        });
+      });
+      test('should ignore root or nested $refs in values of OAS3 media type `examples`', () => {
+        const input = {
+          openapi: '3.0.0',
+          paths: {
+            '/': {
+              post: {
+                requestBody: {
+                  content: {
+                    'application/json': {
+                      schema: {},
```

Comment:
Can we add `example` entry here alongside with `examples` to have test for https://github.com/swagger-api/swagger-js/pull/1810/files#diff-649819c985ad609b5b71b21fc77464a381e6e202379c30f480dab171a19b250fR43 ?

Reply:
@char0n the test for `content.*.example` already exists here:
https://github.com/swagger-api/swagger-js/blob/master/test/bugs/ui-4924.js

Should I move this test to `test/specmap/refs.js` for consistency?

Reply:
@hkosova yes, please. We're moving away from test fragmentation in `test/bugs/` directory.

Reply:
✔️  done